tests/Functional/Repositories/ORM/AccountRepositoryTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\App\Functional\Repositories\ORM;
+
+use App\Repositories\Doctrine\AccountRepository;
+use App\Repositories\Interfaces\AccountRepositoryInterface;
+use Tests\App\Tools\TestCases\AbstractDatabaseTestCase;
+
+/**
+ * @covers \App\Repositories\Doctrine\AccountRepository
+ */
+final class AccountRepositoryTest extends AbstractDatabaseTestCase
+{
+    /**
+     * Test findBySubscriptionType method which is passed by a paramater value not equal to lifetime and monthly.
+     *
+     * @return void
+     */
+    public function testFindBySubscriptionTypeNotLifetime(): void
+    {
+        $repository = $this->app->get(AccountRepositoryInterface::class);
+        $method = $this->getMethodAsPublic(AccountRepository::class, 'findBySubscriptionType');
```

Comment:
No need to get method as public here since `findBySubscriptionType` is public already. Just invoke it directly.